src/objects/config.ts

```diff
@@ -1,9 +1,32 @@
 import { watch } from 'vue'
-import { RunGraphConfig } from '@/models/RunGraph'
+import { RequiredGraphConfig, RunGraphConfig } from '@/models/RunGraph'
 import { EventKey, emitter, waitForEvent } from '@/objects/events'
 import { waitForScope } from '@/objects/scope'
 
-let config: RunGraphConfig | null = null
+let config: RequiredGraphConfig | null = null
+
+const defaults = {
+  styles: {
+    nodeHeight: 20,
```

Comment:
I think previously the height was calculated by the font height + 8px on either side, something like that. A height in the config like this isn't a bad call though. I could see making things more condense with more nodes or something.